.circleci/config.yml

```diff
@@ -83,15 +83,15 @@ jobs:
           name: Generate register descriptors
           command: ./Support/Scripts/generate-register-descriptors.sh
       - *check_dirty_step
-  Documentation:
-    <<: *docker_config
-    environment:
-      ENCRYPTION_LABEL: c06a05533acc
-    steps:
-      - checkout
-      - run:
-          name: Generate documentation
-          command: ./Support/Scripts/generate-documentation.sh
+  #Documentation:
```

Comment:
I don't think you need to comment this block.

Reply:
True! Let me uncomment it before I commit.